### DISCUSSION TITLE
We created a CLIP index for the datacomp-12.8M dataset :mag: :framed_picture:

### DISCUSSION THREAD

**Discussion by RobbeSneyders** [mlfoundations/datacomp#78] [Show and tell]
We created a CLIP index for the datacomp-12.8M dataset to help users search through it and use it more easily. We used [Fondant](https://github.com/ml6team/fondant) to create it and published it on the [huggingface hub](https://huggingface.co/datasets/fondant-ai/datacomp-small-clip). You can find more details and info on how to use it in [this short post](https://fondant.ai/en/latest/blog/2024/03/05/building-a-datacomp-clip-index-with-fondant/).

Let us know what you think, we might create more indices for the larger datasets later.

**Comment by ludwigschmidt** [comment]
Very cool, thanks for sharing! Are you also planning to make indices for the larger pools we have?

**Reply by RobbeSneyders** [reply]
We haven't had the biggest show of interest for the small index yet, so currently we don't see a lot of benefit in making the additional investment. If there's a way to bring this more to the attention of the community, that could tip the scales.
